Userland/Libraries/LibWeb/CodeGenerators/WrapperGenerator.cpp

```diff
@@ -1437,6 +1437,9 @@ static @fully_qualified_name@* impl_from(JS::VM& vm, JS::GlobalObject& global_ob
 )~~~");
         } else if (return_type.name == "EventHandler") {
             scoped_generator.append(R"~~~(
+    if (retval.callback.is_null())
+        return JS::js_null();
```

Comment:
Looks like this object (`EventHandler`) can have a `string` or a `callback`, I'm not sure what the string is for, but it doesn't seem to be handled here at all...

Assuming that's not a problem, please rephrase the commit a bit better, maybe something like "Avoid null deref in impl_from() on empty event handlers"?

Reply:
As far as I could see the `string` was only used as a temporary storage of javascript code which is then converted to a function see: `GlobalEventHandlers::set_event_handler_attribute`. And we were returning only the callback for now anyway.

And whoops I'm too late to change the commit message now

Reply:
Yet another ninja merge :P